lib/matplotlib/axis.py

```diff
@@ -973,6 +979,28 @@ def set_tick_params(self, which='major', reset=False, **kwargs):
 
         self.stale = True
 
+    def get_tick_params(self, which='major'):
+        """
+        Get appearance parameters for ticks, ticklabels, and gridlines.
+
+        Parameters
+        ----------
+        which : {'major', 'minor'}, default: 'major'
+            The group of ticks to which the parameters are applied.
+
```

Comment:
We should add a `Returns` section, stating that it returns a dict, but only contains entries that deviate from the default.